### PR TITLE
Updated README.md to contain the link for the webpage (easier to find).

### PR DESCRIPTION
**Pull request by StayerX**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
 This is the index for the project-renard design notes.
+<http://project-renard.github.io/>
```
Comment (zmughal) [inline README.md]: Please change to `https`. We want to remain secure. :-P

**Comment by StayerX** [comment]
@zmughal done.

**Comment by zmughal** [comment]
Thanks, merging!

**Comment by zmughal** [comment]
Woops, that was off the master branch. It needs to be off the source branch. You should cherry pick your commit and make another pull request.

**Pull request merged**